src/Service/S3/tests/Unit/Result/GetObjectOutputTest.php

```diff
@@ -57,7 +57,8 @@ public function testMetadata()
 
         $metadata = $result->getMetadata();
         self::assertCount(1, $metadata);
-        self::assertArrayHasKey('x-amz-meta-tobias', $metadata);
-        self::assertEquals('nyholm', $metadata['x-amz-meta-tobias']);
+        self::assertArrayNotHasKey('x-amz-meta-tobias', $metadata);
+        self::assertArrayHasKey('tobias', $metadata);
```

Comment:
This change mimics AWS SDK

IMHO `x-amz-meta-` is an internal prefix and shoudn't have leaked to end user. When people use the official SDK to send `tobias`, they expect to get `tobias` in return.

Reply:
Yes, True. I missed this before. 

Thank you